packages/react-data-grid/src/ReactDataGrid.js

```diff
@@ -691,8 +692,9 @@ class ReactDataGrid extends React.Component {
     if (typeof gridWidth === 'undefined' || isNaN(gridWidth) || gridWidth === 0) {
       gridWidth = '100%';
     }
+    const paddingBottom = this.state.editorOpenedAtBoundary ? 300 : 0;
     return (
-      <div className="react-grid-Container" style={{width: containerWidth}}
+      <div className="react-grid-Container" style={{width: containerWidth, paddingBottom}}
```

Comment:
This will only happen if there is custom editor, and the editor is opened within the last 4 rows

https://github.com/adazzle/react-data-grid/blob/9ddc583bd9985dea5f8b6dc2e10578191d2b9738/packages/react-data-grid/src/masks/InteractionMasks.js#L188